### PR TITLE
move test_performance to tools

### PR DESCRIPTION
**Pull request by LHT129**
- split the eval dataset from test_performance

### CODE REVIEW

**Review by wxyucs — APPROVED** [review]
lgtm

File: tools/test_performance.cpp [hunk 1]
```diff
@@ -90,219 +90,6 @@ getIntersection(const int64_t* neighbors,
...
-    int64_t number_of_base_;
-    int64_t number_of_query_;
-    int64_t dim_;
-    size_t train_data_size_;
-    size_t test_data_size_;
-    std::string train_data_type_;
-    std::string test_data_type_;
-};
-
 class Test {
```
Comment (jiaweizone) [inline tools/test_performance.cpp]: change `Test` to `PerfTools` ?

Comment (LHT129) [reply tools/test_performance.cpp]: done

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**